apps/omg_eth/test/support/root_chain_helper.ex

```diff
@@ -24,7 +24,7 @@ defmodule Support.RootChainHelper do
 
   import OMG.Eth.Encoding, only: [to_hex: 1, from_hex: 1]
 
-  @tx_defaults Eth.Defaults.tx_defaults()
+  @tx_defaults OMG.Eth.Defaults.tx_defaults()
```

Comment:
:question:

Reply:
I don't like these partial aliases, because it's difficult to find where a module is used!